rotkehlchen/data_import/importers/rotki_events.py

```diff
@@ -102,15 +104,12 @@ def _import_csv(self, cursor: DBCursor, filepath: Path, **kwargs: Any) -> None:
                         f'During rotki generic events CSV import, found action with unknown '
                         f'asset {e.identifier}. Ignoring entry',
                     )
-                    continue
```

Comment:
This is probably fine but not future-proof. I can see someone adding extra logic to the end of the function and adding a bug that will only hit when an exception hits.

Can you add  comment in the end of the function, that if we ever add more logic to add continue to all exception handlers?